nucliadb_cluster/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tracing = "0.1"
 uuid = "0.8"
 log = "0.4.14"
 env_logger = "0.9.0"
+chitchat = { git = "https://github.com/r3bu1ld3r/chitchat.git", branch="main" }
```

Comment:
If we need a fork we should fork at nuclia org

Reply:
Switched back on quickwit repository